tasks/main.yml

```diff
@@ -57,3 +57,9 @@
     state="started"
     enabled="yes"
     masked="no"
+
+- name: MySQL Upgrade
+  shell: mysql_upgrade
+  async: 45
+  poll: 10
+  when: mysql_upgrade == 1
```

Comment:
Can this just be a boolean variable?  And it should be defaulted somewhere to false.